docs/library/theodore.md

```diff
@@ -0,0 +1,160 @@
+| theodore.cfg | Config file |
+
+### Geometry and timing
+
+- The Theodore core's base width is 672 pixels
+- The Theodore core's base height is 432 pixels
+
+## Usage
+
+![](../image/core/theodore/to8d_os.jpg)
+
+Click the top yellow rectangle on the right-hand side.
+
+## Core options
+
+The Theodore core has the following option(s) that can be tweaked from the core options menu. The default setting is bolded.
+
+- **Floppy write protection** [theodore_floppy_write_protect]  (disabled|**enabled**)
+
+- **Tape write protection** [theodore_tape_write_protect]  (disabled|**enabled**)
+
+- **Dump printer data to file** [theodore_printer_emulation] (**disabled**|enabled)
+
+## User 1 device types
+
+The Theodore core supports the following device type(s) in the controls menu, bolded device types are the default for the specified user(s):
+
+- None - Input disabled
+- **RetroPad** - Joypad
+- RetroPad w/ Analog - Joypad
+
+## Other devices
+
+- Light pen - The TO8 light pen inputs are mapped to the mouse
+
+## Joypad
+
+| RetroPad Inputs                        | User 1 input descriptors    |
+|----------------------------------------|-----------------------------|
+| ![](../image/retropad/retro_a.png)     | "Fire" button               |
+| ![](../image/retropad/retro_b.png)     | Start button                |
+| ![](../image/retropad/retro_x.png)     | Virtual keyboard: go up     |
+| ![](../image/retropad/retro_y.png)     | Virtual keyboard: go down   |
+| ![](../image/retropad/retro_start.png) | Virtual keyboard: keystroke |
+
+On controllers without Y/X keys, select can also be used to roll the virtual keyboard up. The order of the keys in the virtual keyboard is: digits (0->9) then letters (A->Z) then "Enter".
+
+## Keyboard
+
+![](../image/core/theodore/to8d_keyboard.jpg)
+
+| RetroKeyboard Inputs         | Theodore Inputs           |
+|------------------------------|---------------------------|
+| Keyboard Tab                 | STOP                      |
+| Keyboard Left Control        | CNT                       |
+| Keyboard Caps Lock           | CAPSLOCK                  |
+| Keyboard Left Alt            | ACC                       |
+| Keyboard Home                | HOME                      |
+| Keyboard Up                  | UP                        |
+| Keyboard Down                | DOWN                      |
+| Keyboard Right               | RIGHT                     |
+| Keyboard Left                | LEFT                      |
+| Keyboard Insert              | INS                       |
+| Keyboard Delete              | DEL                       |
+| Keyboard F1                  | F1                        |
+| Keyboard F2                  | F2                        |
+| Keyboard F3                  | F3                        |
+| Keyboard F4                  | F4                        |
+| Keyboard F5                  | F5                        |
+| Keyboard F6                  | Shift + F1                |
```

Comment:
It's the other way around for F6->F10: on the TO8, F6->F10 are on the same keys as F1->F5, so for example, on a TO8 keyboard, you have to type "Shift+F1" to do "F6". I translated it the same way on a PC keyboard (because it was easier to manage), so you have to type "Shift+F1" to do "F6" on the TO8.

Reply:
Users do not need that level of detail indeed. Will change "Shift + Fn" by "F(n+5)"

Reply:
reversing that